### PR TITLE
Feature/alphabet sort

### PR DESCRIPTION
**Pull request by arpitgoyal43**
Java program 1

### CODE REVIEW

File: AlphabetOrder.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import java.util.*;
```
Comment (abkesarwani) [inline AlphabetOrder.java]: remove wild card import.

File: AlphabetOrder.java [hunk 2]
```diff
@@ -1,9 +1,13 @@
...
-	{
-		Scanner in =  new Scanner(System.in);
-		System.out.println("Enter any string: ");
\ No newline at end of file
+class AlphabetOrder {
+	public static void main(String args[]) {
+		Scanner in = new Scanner(System.in);
+		System.out.println("Enter any string: ");
+		String str = in.nextLine();
+		char[] temp = str.toCharArray();
```
Comment (abkesarwani) [inline AlphabetOrder.java]: use meaning full variable instead of "temp"

File: AlphabetOrder.java [hunk 3]
```diff
@@ -1,9 +1,13 @@
...
+		Arrays.sort(temp);
```
Comment (abkesarwani) [inline AlphabetOrder.java]: do not use in built method create your own API for this.

Comment (arpitgoyal43) [reply AlphabetOrder.java]: Done

File: AlphabetOrder.java [hunk 4]
```diff
@@ -0,0 +1,36 @@
+import java.util.Scanner;
+import java.util.stream.IntStream;
+
+class AlphabetOrder {
+	public static void main(String args[]) {
+		Scanner in1 = new Scanner(System.in);
+		System.out.println("Enter no. of strings: ");
+		int count = in1.nextInt();
+
+		System.out.println("Enter any string: ");
```
Comment (abkesarwani) [inline AlphabetOrder.java]: remove unused code

File: AlphabetOrder.java [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+					temp = str[i];
+					str[i] = str[j];
+					str[j] = temp;
+				}
+			}
+		}
+		System.out.println("Sorted strings in alphabetical order: ");
+		for(int i=0;i<count;i++)
+		{
+			System.out.print(str[i]+ " ");
```
Comment (abkesarwani) [inline AlphabetOrder.java]: please print output in new line because your string may contains space.

**Review by abkesarwani — APPROVED** [review]
now code looks good :)

File: AlphabetOrder.java [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+		Scanner input1 = new Scanner(System.in);
+		System.out.println("Enter no. of strings: ");
+		int count = input1.nextInt();
+		String[] str = new String[count];
+		Scanner input2 = new Scanner(System.in);
+		System.out.println("Enter the strings to be sorted in a sequence: ");
+		for (int i = 0; i <count; i++) {
+			str[i] = input2.nextLine();
+		}
+		String temp;
```
Comment (SahilKharya) [inline AlphabetOrder.java]: Try a proper naming

**Pull request merged**